app/controllers/logins_controller.rb

```diff
@@ -0,0 +1,69 @@
+class LoginsController < ApplicationController
+  before_action :set_login, only: %i[show edit update destroy]
+
+  # GET /logins or /logins.json
+  def index
+    @logins = Login.all
+  end
+
+  # GET /logins/1 or /logins/1.json
```

Comment:
* Good job implementing some of the views according to the wireframes provided, but in the [project requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Implement%20the%20design,everything%20functions%20first)you are to implement also the login page, Kindly fix this 🙏 below is a screenshot of the view ⬇️ 
![image](https://user-images.githubusercontent.com/52497246/180337990-330d2586-4573-43ad-8c5b-4e4b4ccf1dc9.png)

Reply:
@Davidosky007 Thank tou for your valuable feedback. I will implement recommendations.